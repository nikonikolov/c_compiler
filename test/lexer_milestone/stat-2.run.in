int main()
{
  while(0){}
}

