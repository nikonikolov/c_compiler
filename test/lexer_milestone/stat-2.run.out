int Keyword Tint 1  1
main Identifier TIdentifier 1  1
( Operator T( 1  1
) Operator T) 1  1
{ Operator T{ 2  2
while Keyword Twhile 3  3
( Operator T( 3  3
0 Constant TIntConstant 3  3
) Operator T) 3  3
{ Operator T{ 3  3
} Operator T} 3  3
} Operator T} 4  4
